server.js

```diff
@@ -1832,7 +1833,7 @@ module.exports.insertDevUser = function (callback) {
   });
 };
 
-if (config.startServer) {
+if (config.startServer && require.main === module) {
```

Comment:
I made this change so I could safely require this file without mucking around with the config.